src/QMCApp/qmcapp.cpp

```diff
@@ -121,7 +118,7 @@ int main(int argc, char** argv)
             getwords(words, fin);
             if (words.size())
             {
-              if (words[0].find("xml") < words[0].size())
+              if (words[0].find(".xml") == words[0].size() - 4)
```

Comment:
The nc below also behaves very similar to the 'clones' you just removed and thus I think we should also remove the `nc` handling copies. 

Could you also update https://qmcpack.readthedocs.io/en/develop/running.html?highlight=command%20line#running-qmcpack
`<XML input file(s)>` to `<XML input file(s) and/or text file(s)>` and document the desired text file content.

Reply:
Done